mapiproxy/libmapistore/mapistore_python.c

```diff
@@ -2708,8 +2757,6 @@ static enum mapistore_error mapistore_python_message_set_read_flag(void *message
 		return MAPISTORE_ERR_CONTEXT_FAILED;
 	}
 
-	/* TODO Send Receipt */
```

Comment:
You probably want to check the type of pres here, because PyLong_AsLong() will not return anything predictable otherwise.